hw2/05-form.js

```diff
@@ -0,0 +1,32 @@
+const form = document.querySelector("form");
+const nameInput = document.querySelector("#name");
+const emailInput = document.querySelector("#email");
+const registrationStatusInput = document.querySelector("#registration-status");
+const fullTimeCourseInputs = document.querySelectorAll('input[name="full-time-course"]');
+const additionalInfoInput = document.querySelector("#additional-info");
+
+form.addEventListener("submit", function (event) {
```

Comment:
Airbnb Style guide recommends using Arrow function instead of the "function" keyword.